Microsoft.Toolkit.Uwp.SampleApp/SamplePages/samples.json

```diff
@@ -39,6 +39,7 @@
         "XamlCodeFile": "HamburgerMenuCode.bind",
         "Icon": "/SamplePages/HamburgerMenu/HamburgerMenu.png",
         "BadgeUpdateVersionRequired": "DEPRECATED",
```

Comment:
Should we just automatically add the deprecated badge if we see a DeprecatedWarning?

Reply:
We still need more descriptive text to show the warning, deprecated is not enough I think, that's why I added the new property.